src/coreclr/jit/morph.cpp

```diff
@@ -3779,6 +3786,11 @@ GenTree* Compiler::fgMorphMultiregStructArg(CallArg* arg)
         loadExtent = elems[idx].Offset + genTypeSize(elems[idx].Type);
     }
 
+#ifdef TARGET_LOONGARCH64
+    // For LoongArch64's ABI, the struct {long a; float b;} may be passed
+    // by Integer register and float register while must be considered the padding here.
```

Comment:
```suggestion
    // by integer and float register while must be considered the padding here.
```

> while must be considered the padding here

Can you reword this? It is not exactly clear what you mean here.

Reply:
OK, thanks